cockatrice/src/shortcutssettings.h

```diff
@@ -126,357 +126,405 @@ class ShortcutsSettings : public QObject
 
 private:
     const QChar sep = ';';
-    const QString custom = "Custom";
+    const QString custom = "Custom"; // name of custom group in shortCutsFile
     QString settingsFilePath;
     QHash<QString, ShortcutKey> shortCuts;
 
     QString stringifySequence(const QList<QKeySequence> &Sequence) const;
     QList<QKeySequence> parseSequenceString(const QString &stringSequence) const;
 
-    typedef ShortcutKey SK;
     const QHash<QString, ShortcutKey> defaultShortCuts = {
-        {"MainWindow/aCheckCardUpdates", SK(QT_TRANSLATE_NOOP("shortcutsTab", "Check for card updates"),
-                                            parseSequenceString(""),
+        {"MainWindow/aCheckCardUpdates", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Check for card updates"),
```

Comment:
Did you change anything except of the `SK` -> `ShortcutKey`?
This is quite a pain in the ass to review.

Reply:
I did, erm, a few of them. I can remove that if needed.

Reply:
What i found: 
```
Local gameplay => Start local game
Save deck (clipboard; annotated) => Save deck to clipboard annotated
Save deck (clipboard, raw) => Save deck to clipboard raw
Add/Remove/Set X counter => Add/Remove/Set colorless counter
Move to power (+1/-1) => Move toughness to power (+1/-1)
Move to toughness (-1/+1) => Move power to toughness (-1/+1)
Main 1 => First main phase
Main 2 => Second main phase
Flip card => Turn card over
[move card to..] Top library => Top of library
Graveyard Once => Move to graveyard once
Graveyard Multiple => Move multiple to graveyard
Exile Once => Move to exile once
Exile Multiple => Move multiple to exile
Draw card => Draw a card
Draw cards => Draw multiple cards
Rotate view CW => Rotate view clockwise
Rotate view CCW => Rotate view counterclockwise
``` 

All of them looks fine to me.